### DISCUSSION TITLE
Where is the best place for feature requests?

### DISCUSSION THREAD

**Discussion by thorst** [nextgenhealthcare/connect#4992] [General]
Here on github? In the success center ideas page (https://www.community.nextgen.com/optimization/ideas/ideaList.apexp)? Both?

I have a couple key features needed to make mirth work for our organization that I think others would need as well. One is a testing tool (which Im on the list of beta testers) but there are other things like edit/save/deploy a connector instead of requiring it to be at the channel level (so that multiple devs can be in one channel at a time). Also would be nice to have channel or ideally connector level locking, so when a dev is in there making changes all other devs can only open it in read only. I know a channel is stored as one xml payload in the db, but to modify a connector specifically I would retrieve the xml, sub in the section, and then save it back. We have a lot of interfaces and a lot of devs, and I dont understand how other shops aren't pulling their hair out on stomping on each other. Separating to use channel writers everywhere helps with the issue, but it would seem to be optimal to have it all in one channel...

**Comment by jonbartels** [comment]
> Both?

IMO yes both. Success Center is you as a paying customer. Github/slack/forums are a more conversational arena to work out ideas.

> One is a testing tool (which Im on the list of beta testers) but there are other things like edit/save/deploy a connector instead of requiring it to be at the channel level (so that multiple devs can be in one channel at a time)

This is a common ask. https://github.com/nextgenhealthcare/connect/discussions/4689 is an attempt to consolidate chat, forum, and discussion threads.
